jep/3/README.adoc

```diff
@@ -0,0 +1,191 @@
+= JEP-3: Adding the "Released As" field and "Ready for Release" status to Jenkins JIRA
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 3
+
+| Title
+| Adding the "Released As" field and "Ready for Release" status to Jenkins JIRA
+
+| Sponsor
+| link:https://github.com/oleg-nenashev[@oleg-nenashev]
+
+| Status
+// Uncomment the appropriate line.
+| Draft :information_source:
```

Comment:
This entry should be *Draft Not Submitted*, Draft is the line below.

CC @bitwiseman